CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ if (${CMAKE_CXX_COMPILER_ID} MATCHES "^(Apple)?Clang$")
         unset(CMAKE_EXE_LINKER_FLAGS)
         # This is a bug in CMake that causes it to prefer the system version over
         # the one in the specified ROOT folder.
-        set(OPENSSL_ROOT_DIR ${OPENSSL_ROOT_DIR} /usr/local/Cellar/openssl@1.1/1.1.1g/)
+        set(OPENSSL_ROOT_DIR ${OPENSSL_ROOT_DIR} /usr/local/opt/openssl)
```

Comment:
why?

Reply:
/usr/local/opt/openssl is a soft link, no matter the openssl version changes

Reply:
my main concern is, if in user's env, it points to a new version of openssl which introduces a bug, we have no way to notice that.

Reply:
my main concern is, if user is using a new version of openssl, which introduces a bug, our test has no way to cover that before release.

this is the reason we specify the version of each dependency explicitly.

Reply:
as discussed offline, this is only for mac